### DISCUSSION TITLE
Layout veraltet

### DISCUSSION THREAD

**Discussion by SaschaFroelich** [tud-cd/tudscr#102] [General]
Das Layout ist veraltet und entspricht nicht mehr dem aktuellen Corporate Design der TU Dresden.

**Comment by mrpiggi** [comment]
Korrekt. Leider habe ich aktuell absolut keine Zeit irgendetwas daran zu ändern.

**Reply by SaschaFroelich** [reply]
Ok dann sollte man das Repo in den Ruhestand versetzen und auch die Links auf den Webseiten der TUD entfernen. Es macht keinen Sinn das als offizielles Tex-Repo der TUD weiterzuführen wenn es nicht maintained wird.

**Reply by tpetzoldt** [reply]
Ich finde es sehr wichtig, dass es für aktuelle CD ein LaTeX-Fomat gibt, nicht nur für LaTeX sondern auch für (teilweise) darauf aufbauende Backends, wie z.B. von [Quarto](https://quarto.org/). Die Frage ist natürlich, wer sich darum kümmert. Bis das geklärt ist, ist das veraltete Layout auf jeden Fall besser als wenn jede:r anfängt, eine eigene Titelseite zu basteln.

**Reply by mrpiggi** [reply]
Das ist kein offizielles Angebot sondern in meiner Freizeit entstanden. Weiterhin bin ich auch nicht mehr Mitarbeiter der TUD. 

Falls jemand Änderungswünsche hat, darf sie/er sich gerne mit dem Quelltext auseinandersetzen und einen PR hier platzieren. Bei konkreten Fragen und Problemen stehe ich selbstverständlich wie immer helfend bereit.

**Reply by tpetzoldt** [reply]
Genau, deshalb ein großes Dankeschön an @mrpiggi!

**Reply by SaschaFroelich** [reply]
@tpetzoldt Naja, also so wie es jetzt ist muss sich ja jeder seine eigene Titelseite basteln. Ich habe mich auf das TUD-Latex-Repo verlassen (was offiziell auf der Webseite der TUD verlinkt ist), nur um vom Promotionsamt gesagt zu bekommen, dass das Layout veraltet ist. Und jetzt muss ich eben alles auf den letzten Drücker selbst zusammenklöppeln. Hätte ich das Repo gar nicht erst benutzt, hätte mir das den Stress erspart.

Evtl. sollte man dann irgendwo wenigstens prominent dazuschreiben, dass das Layout nicht mehr aktuell ist.

**Reply by mrpiggi** [reply]
Du kannst jederzeit von tudscrbook auf scrbook wechseln. Formalien solltest du meiner Meinung nach auch nicht direkt vor der Abgabe klären. Und wenn es nur am Titelblatt scheitert, dann kannst du das ja vor dem Binden auch einfach durch eine offizielle Word-Titelseite ersetzen. Klingt jetzt nicht nach so einem Aufwand, als dass man hier seinen Frust abladen müsste

**Reply by SaschaFroelich** [reply]
Bitte sachlich bleiben.

Ja ich schaff das schon, keine Sorge. Ich bleibe aber bei meinem Standpunkt, dass die TUD keine Packages offiziell bewerben sollte, wenn sie veraltet sind und nicht mehr maintained werden. Oder dass es wenigstens irgendwo vermerkt sein sollte.

Alles Gute weiterhin.

**Reply by tpetzoldt** [reply]
@SaschaFroelich: Der Vorteil vom Template ist, dass alle wichtigen Elemente dabei sind. Das ist beim Selbstgebastelten oft nicht der Fall, von Layoutmissgriffen ganz zu schweigen. Klar, auch wir würden es sehr begrüßen, wenn es für Praktikums-, Bachelor oder Masterarbeiten aktuellere Vorlagen gäbe. Solange das noch nicht der Fall ist, empfehle auch ich tud-cd weiter (siehe [https://github.com/tpetzoldt/hyb-tud-thesis-starterkit](https://github.com/tpetzoldt/hyb-tud-thesis-starterkit)). Weitere Dinge gern per E-Mail oder in der Matrix.

**Reply by keinstein** [reply]
@SaschaFroelich Es ist besser, es gibt eine zentrale alte Vorlage, die dann irgendwann zentral angepasst wird, als dass es gar keien Vorlage gibt. Wenn die veraltete Vorlage aus dem Netz genommen wird, wird es wieder taustende individueller und unvollständig umgesetzer Vorlagen geben. Und die auf das korrekte Format zu trimmen ist viel aufwändiger, als die jetzigen Pakete anzupassen. Denn jedes dieser tausenden Pakete hat seine eigene Syntax, und ist meist nur für die Gewohnheiten des Schreibers gemacht. Solche Pakete sind oft schon veraltet, bevor sie bekannt sind. Und dann sind sie noch schneller unbrauchbar as das, was Du hier findest. Wir hatten das schonmal mit dem TUD-beamer-Paket, bevor ich das auf einen Beamer-Stil zurechtgestutzt habe.

Die hier veröffentlichten Pakete haben eine professionelle Qualtiät, die sonst nix an der TU hat und das zu verschrotten halte ich schlichtweg für Ressourcenverschwendung. Wenn Du wissen willst, welcher Wert dahinter steckt, dann lass Dir mal die Arbeitszeit in Stunden nennen, die @mrpiggi  damit verbracht hat und multipliziere das mit 120 €.

Und wenn Du die notwendigen Anpassungen nicht selbst machen möchtest, dann kannst Du auch jemanden beauftragen, diese Leistung zu erbringen. Im Übrigen möchte ich Dich noch darauf hinweisen, dass das Deckblatt für Dissertationen und ähnliche Arbeiten ein Formular ist, das unabhängig vom CD definiert wird.

**Reply by mrpiggi** [reply]
> Bitte sachlich bleiben.

Nun, inwiefern persönliche Befindlichkeiten mit Sachlichkeit einhergehen, müsste ich mir noch einmal erklären lassen.

In dem Bundle steckt viel Herzblut und wahnsinnig viel zeitlicher Aufwand. Dass die Uni sich nach der Umstellung der Schriftart auf Open Sans (die ich noch komplett und weiterhin kompatibel zur Univers implementiert habe) drei, vier Jahre später noch mal dazu entscheidet, abermals das Layout zu ändern, liegt nun mal nicht in meiner Hand. Mittlerweile habe ich aber neben dem Beruf auch noch Familie, da gibt es keine freien Wochenenden zum Coden. Deshalb ist da bisher nicht mehr viel passiert.

Die Förderung, einfach alles einzustampfen habe ich deshalb durchaus persönlich genommen. Sollten deine Statements aber sachliche Beiträge gewesen sein um zu eruieren, warum das Bundle aktuell nicht auf dem letzten Stand des CDs der Uni ist und wie das Problem gelöst werden könnte, bitte ich vielmals um Entschuldigung

**Reply by SaschaFroelich** [reply]
Meine Befindlichkeiten spielen hier keine Rolle. Worauf ich hinaus wollte, und anscheinend habe ich das nicht richtig klar gemacht, ist dass das Repo offiziell von der TUD verlinkt wird, z.B. hier: https://tu-dresden.de/intern/services-und-hilfe/kommunizieren-und-publizieren/cd/vorlagen/druck/latex 

Daher werden Studenten, Doktoranden etc. wie ich in Zukunft weiterhin das Latex-Template benutzen, um dann später erklärt zu bekommen, dass das Layout veraltet ist und sie das wieder ändern müssen. Das macht für mich einfach keinen Sinn. 

Ich stimme zu, dass man das Repo deshalb nicht "einstampfen" muss. Das Mindeste was man aber tun könnte ist, beim o.g. Link und im Repo (am Besten im Readme) dazuzuschreiben, dass es nicht mehr dem aktuellen CD entspricht, im Idealfall sogar noch mit den entsprechenden Änderungen. Evtl. erstelle ich einen PR wenn ich Zeit habe. 

Dass Du da viel Arbeit reingesteckt hast sieht man, und dass die TU seitdem das CD wieder geändert hat, ist bedauerlich. Aber dafür kann ich ja auch nichts.
